CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Version 3.2.1 (Under development)
 
+### App Center
+
+* **[Feature]** Add support for Mac Catalyst.
+* **[Breaking change]** Dropped Xcode 10 support. Xcode 11 is a minimal supported version now.
```

Comment:
Should we move it higher?

Reply:
Shouldn't we use imperative verb here as we do in other lines in the changelog?